heudiconv/convert.py

```diff
@@ -382,9 +387,18 @@ def nipype_convert(item_dicoms, prefix, with_prov, bids, tmpdir):
     convertnode.base_dir = tmpdir
     convertnode.inputs.source_names = item_dicoms
     convertnode.inputs.out_filename = op.basename(op.dirname(prefix))
-    convertnode.inputs.terminal_output = 'allatonce'
     convertnode.inputs.bids_format = bids
-    return convertnode.run()
+    eg = convertnode.run()
+
+    # prov information
+    prov_file = prefix + '_prov.ttl' if with_prov else None
+    if prov_file:
+        safe_copyfile(op.join(convertnode.base_dir,
```

Comment:
so we need a test which runs with prov -- this clause isn't covered

Reply:
yes, but currently i believe this option is broken with latest nipype (#108)

Reply:
so looks like prov <=1.5.0 is fine (nipy/nipype#2240), but we should complain if user's version is higher

Reply:
travis is probably fine/has an old one ;-)